### DISCUSSION TITLE
Seeking Core Maintainers for @CRXJS/vite-plugin

### DISCUSSION THREAD

**Discussion by jacksteamdev** [crxjs/chrome-extension-tools#959] [Announcements]
Hello @CRXJS/vite-plugin community! 

> [!IMPORTANT]  
> CRXJS is still looking for maintainers! Please see #974 if you're interested in joining the team.

Since its launch in 2022, this project has grown beyond my expectations, now helping thousands of developers build Chrome Extensions with modern tools. While the library is feature-complete and stable for most use cases, its growing adoption means it needs more active maintenance than I can provide alone.

## The Current State
- The project is stable and feature-complete for most use cases
- We have a growing user base (3k+ GitHub stars)
- The codebase is mature but complex, handling many edge cases in Chrome Extension development
- Currently, only critical fixes are being merged

## The Path Forward
I'm looking to form a small team of core maintainers (3 positions) to help ensure the long-term sustainability of the project. This is an opportunity to:
- Gain experience maintaining a popular open-source project
- Help shape the future of Chrome Extension development
- Work with modern build tools and browser extensions
- Join a collaborative open-source community

## What We Need
We're seeking 3 maintainers who can commit to:
- Regular PR reviews and issue triage (2-4 hours per week)
- Participating in GitHub Discussions to support users
- Following our established coding standards and review processes
- Working collaboratively with other maintainers

## Required Experience
- Familiarity with Chrome Extension development
- Experience with Vite and modern JavaScript tooling
- Understanding of build tools and bundlers
- Good communication skills for working with the community

## How to Apply
If you're interested in becoming a maintainer:

1. Join our Discord server: https://discord.gg/JbFRSJNp
2. Send a DM to **jacksteamdev** on Discord with:
   - Your experience with Chrome Extensions and build tools
   - Why you're interested in maintaining this project
   - How much time you can commit weekly
   - Links to your relevant work (if any)
3. Create a small PR fixing an existing issue to demonstrate your understanding of the codebase

Applications will be reviewed on an ongoing basis and I'll respond to each candidate individually.

## Next Steps
Once we have our maintainer team in place, we'll:
1. Document maintenance procedures and guidelines
2. Gradually transition responsibilities to the team
3. Establish regular check-ins to ensure smooth operation

Looking forward to working with the new maintenance team to ensure @CRXJS/vite-plugin continues to serve the developer community effectively!

---
Questions? Feel free to ask in the comments below.

**Comment by timsankara** [comment]
Hello,

My name is Tim.

I have experience developing Chrome Extensions and have recently started exploring integrating built-in models within them. Over the past 8 years, I’ve worked as a developer across various projects, which has equipped me with the ability to quickly learn and adapt to new technologies.

I'm interested in maintaining this project because it aligns with my skill set and allows me to contribute to an area I find both technically exciting and impactful. I can commit approximately 8 hours per week, distributed flexibly throughout the week, as this would be a side project for me.

You can review my work on [GitHub](https://github.com/timsankara). While not all projects directly showcase extension development, they demonstrate my technical expertise and problem-solving capabilities.

Looking forward to contributing!

**Reply by jacksteamdev** [reply]
Hi Tim, welcome aboard! ⛵ I've given you Triage access to help with issue management and PR reviews. We'll schedule a maintainer team kickoff after the holidays. Until then, feel free to familiarize yourself with the codebase and ongoing discussions. Happy holidays! 🎄

**Reply by jacksteamdev** [reply]
Feel free to reach out to any developers in your network who might be interested in joining us as maintainers. We're looking for 2 more people to round out the team. Your connections in the Chrome Extension space could be valuable here! 🤝

**Reply by timsankara** [reply]
Glad to be aboard and I'm happy to lend a hand.

**Comment by FliPPeDround** [comment]
Hello @CRXJS/vite-plugin community!

I'm excited about the opportunity to contribute to this fantastic project. Here's a bit about my background and why I'm interested in becoming a maintainer:

### Experience with Vite Plugin Development:
I have a solid understanding of Vite plugin development and am familiar with the Vite ecosystem. My experience with Vite's architecture and plugin system has allowed me to create efficient and useful plugins that enhance the development workflow.

### Experience with Chrome Extensions:
While my primary expertise lies in Vite plugin development, I have also developed several Chrome Extensions using @CRXJS/vite-plugin. This experience has given me a good understanding of the unique challenges and requirements of Chrome Extension development.

### Why I'm Interested:
Open-source has always been a passion of mine. I believe in the power of community-driven projects and the impact they can have on the developer ecosystem. Contributing to @CRXJS/vite-plugin as a maintainer would be a fantastic opportunity to give back to the community, help shape the future of Chrome Extension development, and collaborate with like-minded developers.

### Time Commitment:
I can commit to dedicating 8 hours per week to this project. This time will be spent on PR reviews, issue triage, participating in GitHub Discussions, and collaborating with other maintainers to ensure the project's success.

**Reply by jacksteamdev** [reply]
Welcome, @FliPPeDround! 🎉 Like @timsankara, I've given you Triage access to get started. In the next few days, I'll set up a Discord server so we can coordinate across time zones (I see you're in China!) and arrange our first team meeting.

One more maintainer spot open! 🌟

**Reply by FliPPeDround** [reply]
Thanks! Excited to get started.

**Comment by jacksteamdev** [comment]
## 🎉 Discord Community Launch!

Hey everyone! Quick update on the maintainer search and some exciting news.

First off, I'm happy to welcome @timsankara and @FliPPeDround as our new core maintainers! They've already jumped in to help with issue triage and community support.

To better support our growing community, we've just launched the official CRXJS Discord server! This will be a space for:
- Getting help with CRXJS
- Sharing what you're building
- Connecting with other extension developers
- Real-time discussion with maintainers

Join us at: https://discord.gg/rEpNHh6t

We're still looking for one more maintainer to round out the team. If you're interested, check out the requirements above and drop a comment!

Looking forward to seeing you all in Discord! 🚀

**Reply by jacksteamdev** [reply]
@timsankara and @FliPPeDround On Discord, let's arrange a kickoff meeting at a time that works for everyone.

**Reply by jacksteamdev** [reply]
If your Discord username doesn't match your GitHub username, please DM me on Discord.

**Reply by dsappet** [reply]
Hi @jacksteamdev Thanks for setting up a discord and advancing the community! I've started using crxjs again recently so happy to see it 👍 . 
The discord invite is reported as invalid when I tried to join. Could you update the link?

**Reply by FliPPeDround** [reply]
> Hi @jacksteamdev Thanks for setting up a discord and advancing the community! I've started using crxjs again recently so happy to see it 👍 . The discord invite is reported as invalid when I tried to join. Could you update the link?

https://discord.gg/JbFRSJNp
Welcome!!!

**Reply by jacksteamdev** [reply]
Thanks @FliPPeDround 🚀 

Welcome @dsappet 👋
